connector/mqtt/vernemq/broker/README.md

```diff
@@ -5,22 +5,19 @@ The Dojot VerneMQ service is a extension of [VerneMQ](https://github.com/vernemq
 
 ## **Environment variables**
 
-Key                      | Purpose                                                       | Default Value  | Accepted values
------------------------  | --------------------------------------------------------------| -------------- |------------
-EJBCA_HOSTNAME           | Cluster address                                               | "ejbca-wrapper"| IP or DNSs
-EJBCA_PORT               | Ejbca service port                                            | "5583"         | port values
-USE_VMQ_OPERATOR         | yes if use with vmq-operator                                  | "n"            | y or n
-HOSTNAME                 | Name to container                                             | "broker"       | string
-SERVER_HOSTNAME          | Server hostname (the host to connect)                         | "localhost"    | hostname
-CA_NAME                  | CA Name from EJBCA                                            | "IOTmidCA"    | string
-CHECK_EXPIRATION_TIME    | Checks if the certificates expires every define time by cron  | "0 1 * * *" | cron schedule expressions
+Key                      | Purpose                                                            | Default Value  | Accepted values
+-----------------------  | --------------------------------------------------------------     | -------------- |------------
+EJBCA_HOSTNAME           | Hostname of the EJBCA broker                                       | "ejbca-wrapper"| IP or DNSs
+EJBCA_PORT               | Ejbca service port                                                 | "5583"         | port values
+USE_VMQ_OPERATOR         | yes if use with vmq-operator                                       | "n"            | y or n
```

Comment:
Can it be a little bit more descriptive? like "configuration for using vmq-operator."